examples/external-roles/main.tf

```diff
@@ -0,0 +1,14 @@
+module "external-roles" {
+  source = "../../modules/external-roles"
+
+  roles_map = {
+    "terraform@tf-root-project.iam.gserviceaccount.com" = {
+      "ecosystem-staging-ec90" = [
+        "role/editor",
+      ],
+      "ecosystem-cnp-staging-e1f7" = [
+        "roles/compute.admin",
+      ]
+    }
+  }
+}
```

Comment:
https://github.com/extenda/tf-module-gcp-project/blob/11250639a16c0d7192229adc0669fde949a68dd0/examples/external-roles/provider.tf#L11-L21
why do you need this block for the example?